docs/erd.md

```
@@ -16,423 +23,264 @@
-| `orders`    | `idx_orders_status`       | status           | 상태별 조회      |
-| `wishlists` | `idx_wishlists_user`      | user_id          | 사용자 찜 조회   |
+- users(email)
+- stores(region, status)
+- products(store_id, status, end_at)
+- orders(user_id, store_id, created_at)
 
 ---
 
 # 4. 관계 요약
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)